pkg/kp/rcstore/consul_store.go

```diff
@@ -66,59 +91,42 @@ func (s *consulStore) Create(manifest pods.Manifest, nodeSelector labels.Selecto
 		Disabled:        false,
 	}
 
-	err = s.forEachLabel(rc, func(id, k, v string) error {
-		return s.applicator.SetLabel(labels.RC, id, k, v)
-	})
-
-	// TODO: If the `put` operations fail, we have already labeled the RC,
-	// yet the RC will not exist in the backing Consul KV store.
-	// The labels may have to be cleaned up.
-
-	err = s.put(id, []kvPair{
-		kvPair{key: "disabled", value: []byte("false")},
-		kvPair{key: "node_selector", value: []byte(nodeSelector.String())},
-		kvPair{key: "pod_labels", value: []byte(podLabels.String())},
-		kvPair{key: "pod_manifest", value: buf.Bytes()},
-		kvPair{key: "replicas_desired", value: []byte("0")},
-	})
-
+	jsonRC, err := json.Marshal(rc)
 	if err != nil {
-		return fields.RC{}, err
+		return fields.RC{}, nil
```

Comment:
so this is deliberately not returning the error, when there is one. is it intentional?